OpenSCToken/Token.m

```diff
@@ -116,8 +116,9 @@ - (nullable instancetype)initWithSmartCard:(TKSmartCard *)smartCard AID:(nullabl
         struct sc_pkcs15_cert_info *cert_info = objs[i]->data;
         struct sc_pkcs15_cert *cert = NULL;
         struct sc_pkcs15_object *prkey_obj = NULL;
+        int private_obj = objs[i].flags & SC_PKCS15_CO_FLAG_PRIVATE;
```

Comment:
having second look it should have been probably been `objs[i]->flags`. I opened a new PR with the CI to get some feedback on the changes rather sooner: #47.